farm/modeling/optimization.py

```diff
@@ -1,9 +1,14 @@
 from importlib import import_module
 import logging
+import sys
+import inspect
 
 import torch
-import torch.distributed as dist
 from torch.nn.parallel import DistributedDataParallel
+from torch.nn import DataParallel
+
+# Used indirectly in _get_optim() to avoid name collision with torch's AdamW
+from transformers.optimization import AdamW as TransformersAdamW
```

Comment:
Any reason you don't default to torch's AdamW when available and otherwise use the transformer implementation? They do have a slightly different signature though. (Sorry for lurking.)

Reply:
Sorry, for this late reply! I didn't see your comment here and I was busy with other topics the last weeks. I haven't investigated the differences between AdamW in transformers and pytorch  in detail. One difference however is the `correct_bias` option. This is only implemented in transformers and should be set to `False` for the original BERT implementation. I stumbled across it when trying to replicate some of our earlier experiments on CONLL using this branch here. Interestingly, it had quite some big effect on the early steps of training. So for the sake of reproducibility / backwards compatibility, I would suggest using AdamW from transformers as a default in FARM.